src/Servers/IIS/IIS/test/IIS.ForwardsCompatibility.FunctionalTests/IIS.ForwardsCompatibility.FunctionalTests.csproj

```diff
@@ -24,8 +24,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="AspNetCoreModule" Condition="'$(OS)' == 'Windows_NT'" />
```

Comment:
Doesn't this break tests? I thought we needed local copies of ancm bits here.

Reply:
IIS.IntegrationTesting package would bring required native dependencies.

Reply:
How? Did you add something to make the native assets get copied transitively? Or is the SDK automatically handling transitive project refs to native projects?

Reply:
It was always working because of https://github.com/aspnet/AspNetCore/blob/master/src/Servers/IIS/IntegrationTesting.IIS/src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS.csproj#L39-L43

Reply:
That target ensures the native libraries are copied into the build output of Microsoft.AspNetCore.Server.IntegrationTesting.IIS, but how does this make it into the build output of projects which reference Microsoft.AspNetCore.Server.IntegrationTesting.IIS? I don't see anything that would copy the native libraries into the build output of IIS.ForwardsCompatibility.FunctionalTests.

Or,is this just another case of a project that didn't need to have ancm binaries in build output in the first place?

Reply:
As far as I remember `<None` item group was copied to output directories of projects referencing `Microsoft.AspNetCore.Server.IntegrationTesting.IIS`. We had this working long before `<NativeProjectReference` came along